configs/projects/puppet-agent.rb

```diff
@@ -92,6 +92,7 @@
   proj.component "puppet"
   proj.component "facter"
   proj.component "hiera"
+  proj.component "leatherman"
   proj.component "marionette-collective"
   proj.component "cpp-pcp-client"
   proj.component "pxp-agent"
```

Comment:
It wouldn't hurt to include them, it'd ensure the correct ordering and - as there are no changes to Facter - may avoid needing future changes to puppet-agent to enable using unvendored Leatherman.